terraform/python/k8s/cleanup/main.tf

```diff
@@ -45,6 +45,10 @@ resource "null_resource" "cleanup" {
       # Print cluster state when done clean up procedures
       echo "LOG: Printing cluster state after cleanup"
       kubectl get pods -A
+
+      # Delete ssm parameter for remote service ip
+      aws ssm delete-parameter --name python-remote-service-ip-${var.test_id}
+
```

Comment:
Is this needed? Each test should just override the previous one's value. This doesn't create a failure point but it is unneeded I think

Reply:
Yes this is needed. When we test K8s on different ec2, since we are using the same parameter `python-remote-service-ip` it sometimes overrides the canary's k8s and causes it to failure. Need to have unique parameter for each run and also clean up afterwards

Reply:
Oh I see, didn't notice the SSM parameter is now unique